docs/src/pages/guides/typescript.md

```diff
@@ -94,6 +94,44 @@ const DecoratedNoProps = decorate<{}>( // <-- note the type argument!
 
 To avoid worrying about this edge case it may be a good habit to always provide an explicit type argument to `decorate`.
 
+### Injecting Multiple Classes
+
+TypeScript does not support variadic types, therefore there is no clean way to type usage of `withStyles` that injects multiple classes into a component. However, there is a hacky workaround that allows you to get support for classes inside your component while only exposing the props you define.
```

Comment:
I don't understand this paragraph, or to the extent that I do, I don't think it's accurate. It's certainly possible to inject multiple classes into a component.